docs/tips-and-tricks.md

```diff
@@ -0,0 +1,35 @@
+<p align="center">
+  <h2 align="center">📖 Tips and Tricks for writing a userstyle</h2>
+</p>
+
+&nbsp;
+
+### Live reloading
+
+If you want to see your changes in real-time while using an external editor:
+
+1. Head to `chrome://extensions/` or your browser's equivalent.
+2. Locate the stylus extension and click on the `Details` button.
+3. Enable `Allow access to file URLs`.
+4. Now you can open the `catppuccin.user.css` file in your browser by typing `file:///path/to/catppuccin.user.css` in the address bar.
+5. Then you should notice stylus will load on that page, at this point ensure `live preview` is enabled.
+6. Now you can make changes in your preferred editor and see real-time changes.
+
+> [!TIP]
+> Further reading: [Stylus Wiki (Initial installation and live reload)](https://github.com/openstyles/stylus/wiki/Writing-UserCSS#initial-installation-and-live-reload)
+
+### Hiding sensitive information in screenshots
+
+When taking screenshots of userstyles, you may want to hide sensitive information such as your username, email, or other personal information.
+
+To do this, you we usually use the [Flow Circular](https://fonts.google.com/specimen/Flow+Circular) font to redact the details.
+
+Use this snippet at the top of your userstyle to import and use the font:
+
+```less
+@import url('https://fonts.googleapis.com/css2?family=Flow+Circular&display=swap');
+
+* {
+  font-family: 'Flow Circular', cursive;
+}
+```
```

Comment:
Hmm. Personally I have a separate userstyle that runs on all pages that I can toggle on and off separately:

```less
/* ==UserStyle==
@name           Flow Circular Font
@namespace      flow-circular
@version        1.0.0
@author         uncenter
==/UserStyle== */
@-moz-document regex("*") {
    @import url('https://fonts.googleapis.com/css2?family=Flow+Circular&display=swap');

    * {
        font-family: 'Flow Circular', cursive;
    }
}
```

Reply:
Thats fair but i think its fine to leave this as is, users can do whatever they think works.